doc/src/main/docs/generated/api/response/api/v2/{project}/webrootfield"/{fieldName}/{path}"/404/request-schema.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Why are there quotes in the directory names?
It is impossible to check this out on Windows, because Windows filesystem does not allow quotes